Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerMentionCellView_Tests.swift

```diff
@@ -0,0 +1,234 @@
+    }
+
+    func test_appearanceCustomization_usingAppearanceHook_whenUserOffline() {
+        class TestView: ChatMessageComposerMentionCellView {}
+        TestView.defaultAppearance {
+            // Modify appearance
+            $0.backgroundColor = .white
+            $0.usernameLabel.textColor = .blue
+            $0.usernameTagLabel.textColor = .cyan
+            $0.textStackView.distribution = .equalSpacing
+
+            // Modify layout
+            NSLayoutConstraint.activate([
+                $0.textStackView.topAnchor.constraint(equalTo: $0.topAnchor, constant: 10),
+                $0.textStackView.trailingAnchor.constraint(equalTo: $0.mentionSymbolImageView.leadingAnchor),
+                $0.textStackView.leadingAnchor.constraint(equalTo: $0.avatarView.trailingAnchor, constant: 70),
+                $0.bottomAnchor.constraint(equalTo: $0.textStackView.bottomAnchor, constant: 40),
+                $0.avatarView.widthAnchor.constraint(equalToConstant: 80)
+            ])
+        }
+
+        let view = TestView().withoutAutoresizingMaskConstraints
+        view.widthAnchor.constraint(equalToConstant: Self.defaultCellWidth).isActive = true
+        view.content = ("Mr Always LightVader", "darkforce37", TestImages.vader.url, false)
+        AssertSnapshot(view, variants: [.small])
+    }
+
+    func test_appearanceCustomization_usingAppearanceHook_whenUserOnline() {
+        class TestView: ChatMessageComposerMentionCellView {}
+        TestView.defaultAppearance {
+            // Modify appearance
+            $0.backgroundColor = .white
+            $0.usernameLabel.textColor = .blue
+            $0.usernameTagLabel.textColor = .cyan
+            $0.textStackView.distribution = .equalSpacing
+
+            // Modify layout
+            NSLayoutConstraint.activate([
+                $0.textStackView.topAnchor.constraint(equalTo: $0.topAnchor, constant: 10),
+                $0.textStackView.trailingAnchor.constraint(equalTo: $0.mentionSymbolImageView.leadingAnchor),
+                $0.textStackView.leadingAnchor.constraint(equalTo: $0.avatarView.trailingAnchor, constant: 70),
+                $0.bottomAnchor.constraint(equalTo: $0.textStackView.bottomAnchor, constant: 40),
+                $0.avatarView.widthAnchor.constraint(equalToConstant: 80)
+            ])
+        }
+
+        let view = TestView().withoutAutoresizingMaskConstraints
+        view.widthAnchor.constraint(equalToConstant: Self.defaultCellWidth).isActive = true
+        view.content = ("Mr Always LightVader", "darkforce37", TestImages.vader.url, true)
+        AssertSnapshot(view, variants: [.small])
+    }
+
+    func test_appearanceCustomization_usingSubclassing_whenUserOnline() {
+        class TestView: ChatMessageComposerMentionCellView {
+            override func setUpAppearance() {
+                usernameLabel.textColor = .black
+                usernameTagLabel.textColor = .darkGray
+                backgroundColor = .white
+            }
```

Comment:
You should call `super.setUpAppearance()`

Reply:
Added.